lib/src/table/safe-html.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { DomSanitizer, SafeHtml } from "@angular/platform-browser";
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({ name: "safeHtml" })
+export class SafeHtmlPipe implements PipeTransform {
+    constructor(private sanitizer: DomSanitizer) {}
+    transform(value: string): SafeHtml {
+        if (value && value.length) {
+            return this.sanitizer.bypassSecurityTrustHtml(value);
+        } else {
+            return "";
+        }
+    }
+}
```

Comment:
can you place this pipe outside the component folder, similar to what we have with React. Maybe a folder called `__utils`